t/porting/libperl.t

```diff
@@ -56,7 +56,7 @@ if ($Config{cc} =~ /g\+\+/) {
 }
 
 # ccname is gcc for both gcc and clang
-if ($Config{ccname} eq "gcc" && $Config{ccflags} =~ /-flto\b/) {
+if ($Config{ccname} eq "gcc" && "$Config{ccflags} $Config{optimize}" =~ /-flto\b/) {
```

Comment:
Related to James' point, if he'd used `-flto=N` or `-flto=jobserver` (or `-flto=auto`), his warnings would be gone, but it wouldn't match this regex because of the `\b`.

(For Clang, there's also `-flto=thin`.)

Reply:
Yes, sorry! Quite right.